src/pages/packet/components/packet-row.jsx

```diff
@@ -0,0 +1,30 @@
+import PacketStruct from '../../../data/packet-structure.json';
+import PackageComponent from './package-component';
+import React from 'react';
+
+const PackageRow = ({ trail, build, engage, recruit, TRAILS_LIST, removeOptionChecked, setTrail, isMobile }) => {
```

Comment:
I don't think you need to pass ismobile down in this one, you can make the actual check in here

Reply:
Why? (not questioning your decision, just curious about the reasoning and if this is a ~react thing~)

Also, by "make the actual check" do you mean writing the prop in here too?